pom.xml

```diff
@@ -112,7 +112,7 @@
         <asm.version>9.6</asm.version>
         <jackson.version>2.16.1</jackson.version>
         <log4j2.version>2.23.1</log4j2.version>
-        <ognl.version>3.3.4</ognl.version>
+        <ognl.version>3.3.4-atlassian-1</ognl.version>
```

Comment:
To be replaced with 3.3.5 once released [OGNL patch PR](https://github.com/orphan-oss/ognl/pull/263)

Reply:
Try you to use 3.3.5 verision

Reply:
Updated. Thanks